panel/src/components/Forms/Input/RadioInput.vue

```diff
@@ -41,7 +39,7 @@ export default {
 					disabled: this.disabled,
 					info: option.info,
 					label: option.text,
-					name: this.name,
+					name: this.name ?? this.id,
```

Comment:
The radios and checkboxes won't work properly without a name. That's why it's better to fall back to the id.